src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const splittedArr = sourceString.split(';');
+
+  for (let i = 0; i < splittedArr.length; i++) {
+    const style = splittedArr[i];
+
+    if (style) {
+      const colonIndex = style.indexOf(':');
+
+      if (colonIndex > -1) {
+        const prop = style.slice(0, colonIndex).trim();
+        const value = style.slice(colonIndex + 1).trim();
+
+        result[prop] = value;
+      }
+    }
+  }
```

Comment:
The use of a for loop is not recommended according to the task's checklist. Instead, consider using array iteration methods like 'forEach', 'map', or 'reduce' to process the style declarations.